src/integration-tests/README.md

```diff
@@ -12,3 +12,12 @@ To run the tests:
 * run the tests
     * `cd src/integration-tests`
     * `./run-tests [extra pytest options]`
+* you might also want to specify custom binary locations as follows
+    * `BLAZINGMQ_BUILD_DIR` - the root directory where the resulting binaries reside;
+       default: `cmake.bld/{platform.system()}`
```

Comment:
We use this dir when build with `vcpkg` I believe.
When we build via `bin/build-darwin.sh` or `bin/build-ubuntu.sh`, the directory is `build/blazingmq`
Does it worth it to mention it here?
```suggestion
       default (vcpkg): `cmake.bld/{platform.system()}`
       default (bin scripts): `build/blazingmq`
```

Reply:
The value is taken from [here](https://github.com/bloomberg/blazingmq/blob/main/src/python/blazingmq/dev/paths.py#L15), so building with "bin" scripts is one of the situations when this variable is required.